src/Utilities/OptimizerHacks.cpp

```diff
@@ -8,3 +8,16 @@ namespace optimizer_hacks {
 void indicate_value_can_be_changed(void* /*variable*/) {}
 }  // namespace optimizer_hacks
 #endif  /* defined(__clang__) && __clang__ < 11 */
+
+#if defined(__clang__) && __clang_major__ >= 15
+// clang v15+ fails to instantiate the following std library templates if
+// pre-compiled headers are used
+template std::__cxx11::basic_string<char, std::char_traits<char>,
```

Comment:
`#include <string>` (Have you tested this with the PCH disabled?)

Might be worth putting this behind some combination of `defined(__GLIBCXX__)` and whether the PCH is enabled.